### ISSUE TITLE
Add code coverage

### ISSUE THREAD

**Issue by sdhutchins** [vallenderlab/MicrobiomeR#60]
Use the [covr](https://github.com/r-lib/covr) package to add code coverage badge/stats to package.

- [x] Setup code coverage with usethis `usethis::use_coverage()`
- [x] Add code coverage badge
- [x] Add code coverage commands to `.travis.yml`

```yaml
r_github_packages:
  - r-lib/covr

after_success:
  - Rscript -e 'covr::codecov()'
```

**Comment by sdhutchins** [comment]
Our current code coverage is below...

![image](https://user-images.githubusercontent.com/20976111/51141212-2a4fea80-180e-11e9-9f71-2b6520e5f587.png)

**Comment by sdhutchins** [comment]
The travis build is not generate code coverage.

![image](https://user-images.githubusercontent.com/20976111/52604702-9cc5e180-2e31-11e9-84dc-1142cf25b513.png)

**Comment by sdhutchins** [comment]
Fixed in #78
